frameworks/flaml/setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+HERE=$(dirname "$0")
+#AMLB_DIR="$1"
+# by passing the module directory to `setup.sh`, it tells it to automatically create a virtual env under the current module.
+# this virtual env is then used to run the exec.py only, and can be configured here using `PIP` and `PY` commands.
+. $HERE/../shared/setup.sh $HERE
+#. $AMLB_DIR/frameworks/shared/setup.sh $HERE
+PIP install  flaml==0.2.3
```

Comment:
see comment in `frameworks.yaml` and see `RandomForest/setup.sh` for a template supporting multiple versions.
Maybe we will find a way to consolidate this later.

Reply:
Revised accordingly. Thanks for the pointer.